### ISSUE TITLE
Raspberry Pi's Raspbian

### ISSUE THREAD

**Issue by ramonhugo** [nicehash/nheqminer#187]
Is it possible to compile it in **Raspbian**?
At the moment I'm having an issue in the **assemble.sh**.

**Comment by chronosek** [comment]
it is possible, but you have to drop any function with assembler code, cause it was wrote for x86_64 and will not work for arm cpu, so no xenoncat, but cuda_trump could work... btw you should not expect get even 1 sol/s

**Comment by ramonhugo** [comment]
But to be able to do that I need the CUDA toolkit installed.
I've tried to use the Ubuntu 14.04 CUDA repository, but CUDA needs nvidia drivers.
Are you sure that makes sense to try that?

**Comment by chronosek** [comment]
for my opininon there is no sense, only just if you have too much time....  You would need to change all cmake commands just to skip many parts, you do not need cuda or opengl for raspberry

**Comment by ramonhugo** [comment]
I have several Raspberry Pi 3 mining feathercoin just for fun. But I'd guess would be more profitable to deal with nicehash. Just wanted to test, but seems too much effort.

**Comment by alexellis** [comment]
Any further thoughts on this? cpuminer works well on the Pi, but there is no solver there for equihash.

**Comment by ramonhugo** [comment]
Yes, I'm using **cpuminer** (**minerd**) on **Raspbian** too, but I couldn't make the **nheqminer** to work.
To say "that is not worth it" is dumb, I'm not asking about opinions here.
I can mine **Feathercoin** and **Maxcoin** on **Raspberry P**i, but to mine **Zcash** will be better.
I coudn't make any **Nicehash** miner to work - it seems that they are focused on **Windows**, and even the version of **nheqminer** on **Windows** recognizing my **Intel HD 4400** to mine, the **CPU+GPU** of that is worst (yes, I run two separated threads because the **CPU+GPU** is not working well) in **Windows** than just **CPU** on **Linux**, what is very weird since the developers seems to be focusing on **Windows**.

**Comment by luisdanielmesa** [comment]
they are focused on windows to harness a little botnet power...
